mesonbuild/mparser.py

```diff
@@ -94,6 +94,9 @@ def __eq__(self, other: object) -> bool:
             return self.tid == other.tid
         return NotImplemented
 
+
+IDENT_RE = re.compile('[_a-zA-Z][_0-9a-zA-Z]*')
```

Comment:
```suggestion
_IDENT_RE = re.compile('[_a-zA-Z][_0-9a-zA-Z]*')
```
Might as well mark this as a module implementation detail while we're at it.

Reply:
It's not - it's actually used by `set_variable()`, hence why I extracted it out of the parser.

Reply:
Ah... okay.